umpf

```diff
@@ -341,6 +360,10 @@ find_branch_rev() {
 	local name branch remote
 	local -a branches replies
 	name="${1}"
+	if [ -n "${OVERRIDES[$name]}" ]; then
+	        reply="$(${GIT} rev-parse "${OVERRIDES[$name]}^{}" 2> /dev/null)"
```

Comment:
`${OVERRIDES[$name]}` is already the result of rev-parse. No need to do this again.

Reply:
Fixes.